split-webapp/split/src/pages/Login.js

```diff
@@ -23,6 +23,16 @@ class Login extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const inputField = document.getElementById("inner-container-id");
+    inputField.addEventListener("keyup", function(event) {
+      if (event.keyCode === 13) {
+        event.preventDefault();
+        document.getElementById("login-button").click();
```

Comment:
Would using refs be safer over `getElementById`? Or do you reckon it's fine.

Reply:
It should be fine as no other component has an id field (that i've found)

Reply:
Sweet